client/graphite/read.go

```diff
@@ -296,9 +297,10 @@ func (c *Client) fetchData(ctx context.Context, queryResult *prompb.QueryResult,
 		case ts := <-output:
```

Comment:
We don't need the select anymore, right ?

Reply:
Looks better to me than a for loop

Reply:
ok